tls/_common/_constructs.py

```diff
@@ -149,7 +149,7 @@ def EnumClass(type_field, type_enum):  # noqa
     return construct.SymmetricMapping(type_field, mapping)
 
 
-def EnumSwitch(type_field, type_enum, value_field, value_choices):  # noqa
+def EnumSwitch(type_field, type_enum, value_field, value_choices, default=construct.Switch.NoDefault):  # noqa
```

Comment:
I think I added `# noqa` so that we could follow construct's NamingConventionForFunctions without upsetting pylint.  Two questions:  

1) Maybe it's better to rename these _following_the_standard_convention?  (This would be in a separate PR)
2) Do you think it makes sense to wrap the signature at 79 characters to stay consistent with the rest of the project?